### PR TITLE
Trackers constant collector

### PR DESCRIPTION
**Pull request by costaconrado**
This PR introduces a new collector for PS_Constant project.

In order to achieve that this PR add a new environment variable into github secrets and docker-compose.
This PR also adds new models to OSIDB to store data pulled from the collector.

Closes OSIDB-1199.

### CODE REVIEW

File: collectors/ps_constants/tasks.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    # Execute this every 6 hours
+    # TODO: crontab seems to be not sufficient as a scheduler here
+    # since it is only capable of running the job at every fixed third hour
+    # eg. 3:00,8:00,etc. and thus there exist a scenario in which
+    # the OSIDB is run lets say 3:01 and this job will be scheduled on 8:49
+    # which is really not what we want, since there may be other collectors
+    # depending on this one, odd minute number was chosen in order to not have multiple
+    # tasks running at the same time
+    # TODO: Use django_celery_beat which has PeriodicTask with IntervalSchedule
+    #  What we use here is equivalent to PeriodicTask with CrontabSchedule
```
Comment (osoukup) [inline collectors/ps_constants/tasks.py:43]: This is a known issue (it probably even has a Jira) and can make problems during the very initial sync. The set collect period or the timing should be ideally ignored at such situation and everything should be synced immediately going from the models no dependencies and progressing with others when their dependencies get resolved. If we only have time for it :smile: (not complaining about your code here, so you can ignore this note)

File: collectors/ps_constants/tasks.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    crontab=crontab(minute="49", hour="*/5"),
+    data_models=[SpecialConsiderationPackage, UbiPackage],
+)
+def ps_constants_collector(collector_obj) -> None:
```
Comment (osoukup) [inline collectors/ps_constants/tasks.py:47]: nitpick: a string is being returned, not `None`

File: docs/CHANGELOG.md [hunk 3]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+### Added
+- Implement collector for ps_constant project (OSIDB-1199)
```
Comment (osoukup) [inline docs/CHANGELOG.md:9]: nitpick: the project is I think called `ps-constants` (https://git.prodsec.redhat.com/devops/ps-constants/)

**Comment by costaconrado** [comment]
I have submitted the changes requested, thank you.

**Pull request merged**